ocs_ci/deployment/baremetal.py

```diff
@@ -190,6 +191,21 @@ def deploy_prereq(self):
                 assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Download required File"
             else:
                 raise RhcosImageNotFound
+
+            if ocp_version == "4.6":
```

Comment:
shouldn't this be `if ocp_version >= "4.6":`

Reply:
here we are not sure how it's going to be in 4.7 so I have used only 4.6 if I get that it will be the same will use 4.6 or greater release

Reply:
You cannot compare >= with string @ebenahar

Reply:
> You cannot compare >= with string @ebenahar

I thought so as well but apparently you can:

```
In [1]: version = "4.5"

In [2]: version >= "4.6"
Out[2]: False

In [3]: version <= "4.6"
Out[3]: True
```

Reply:
It is [possible to compare strings](https://docs.python.org/3/reference/expressions.html#value-comparisons), but generally it is probably not good approach (in the meaning, that it might not return expectable result):
```python
In [1]: "a" < "Z"                                                                                                                                
Out[1]: False

In [2]: "A" < "z"                                                                                                                                
Out[2]: True
```
But in this particular usecase, I think it is quite safe to use it this way.

Reply:
Do I need to change anything here or we are good to merge the PR

Reply:
@petr-balogh @dahorak ^^

Reply:
you have == so you are fine from the start of the conversation here ;)

Reply:
Thanks